crates/solidity/outputs/cargo/crate/src/generated/language.rs

```diff
@@ -5605,8 +5605,14 @@ impl Language {
                 choice.finish(input)
             })
         };
-        let primary_expression_parser =
-            |input: &mut ParserContext<'_>| self.version_pragma_specifier(input);
+        let primary_expression_parser = |input: &mut ParserContext<'_>| {
```

Comment:
nit: shouldn't codegen remove the `ChoiceHelper` if it has only one element?

Reply:
I'll follow up in a PR addressing #638, we need to clean this up a bit anyway